icu4c/source/common/uloc.cpp

```diff
@@ -598,7 +592,7 @@ compareKeywordStructs(const void * /*context*/, const void *left, const void *ri
     return uprv_strcmp(leftString, rightString);
 }
 
-static void
+void U_EXPORT2
```

Comment:
U_CFUNC, and maybe not U_EXPORT2. Be consistent decl vs impl.

Reply:
Done.